language/pl-PL/com_users.ini

```
@@ -4,119 +4,159 @@
 COM_USERS_REMIND_REQUEST_ERROR="Błąd żądania przypomnienia hasła."
 COM_USERS_REMIND_REQUEST_FAILED="Przypomnienie nie powiodło się: %s"
-COM_USERS_REMIND_REQUEST_SUCCESS="Wiadomość z przypomnieniem została wysłana pomyślnie. Sprawdź twój e-mail."
+COM_USERS_REMIND_REQUEST_SUCCESS="Wiadomość z przypomnieniem została wysłana pomyślnie. Sprawdź swoją skrzynkę pocztową."
 COM_USERS_REMIND_SUPERADMIN_ERROR="Super użytkownik nie może prosić o przypomnienie hasła. Skontaktuj się z innym super użytkownikiem lub skorzystaj z alternatywnej metody odtwarzania hasła."
-COM_USERS_RESET="Ustawienie hasła"
+COM_USERS_RESET="Zresetowanie hasła"
 COM_USERS_RESET_COMPLETE_ERROR="Błąd podczas dokończenia procesu zmiany hasła."
 COM_USERS_RESET_COMPLETE_FAILED="Ukończenie zmiany hasła nie powiodło się: %s"
-COM_USERS_RESET_COMPLETE_LABEL="Aby zakończyć proces zmiany hasła, podaj swoje nowe hasło."
-COM_USERS_RESET_COMPLETE_SUCCESS="Hasło zostało pomyślnie zmienione. Możesz się teraz zalogować na stronie."
+COM_USERS_RESET_COMPLETE_LABEL="Aby zakończyć proces resetowania hasła, wprowadź nowe hasło."
+COM_USERS_RESET_COMPLETE_SUCCESS="Hasło zostało pomyślnie zresetowane. Możesz teraz zalogować się na stronie internetowej."
 COM_USERS_RESET_COMPLETE_TOKENS_MISSING="Zresetowanie Twojego hasła nie powiodło się, ponieważ nie znaleziono pasującego kodu weryfikacyjnego."
 COM_USERS_RESET_CONFIRM_ERROR="Błąd podczas potwierdzania zmiany hasła."
-COM_USERS_RESET_CONFIRM_FAILED="Potwierdzenie zmiany hasła nie powiodło się ponieważ kod weryfikacyjny jest błędny. %s"
+COM_USERS_RESET_CONFIRM_FAILED="Potwierdzenie zmiany hasła nie powiodło się ponieważ kod weryfikacyjny był nieprawidłowy. %s"
 COM_USERS_RESET_CONFIRM_LABEL="Wiadomość została wysłana na Twój adres e-mail. Wiadomość zawiera kod weryfikacyjny, wklej ten kod weryfikacyjny w polu poniżej, aby udowodnić, że jesteś właścicielem tego konta."
-COM_USERS_RESET_REQUEST_ERROR="Wystąpił błąd podczas prośby o zmianę hasła."
+COM_USERS_RESET_REQUEST="Jeśli wprowadzony adres e-mail jest zarejestrowany na tej stronie internetowej, wkrótce otrzymasz wiadomość e-mail z linkiem do zresetowania hasła do konta."
+COM_USERS_RESET_REQUEST_ERROR="Wystąpił błąd podczas resetowania hasła."
 COM_USERS_RESET_REQUEST_FAILED="Niepomyślna zmiana hasła: %s"
-COM_USERS_RESET_REQUEST_LABEL="Nie pamiętasz hasła? To nie jest problem. Wpisz poniżej swój adres poczty elektronicznej podany podczas rejestracji i kliknij przycisk &bdquo;Wyślij&rdquo;. Otrzymasz specjalny kod umożliwiający zalogowanie się i dokonanie zmiany hasła.<br />Sprawdź za kilka chwil swoją skrzynkę pocztową. Gdy otrzymasz wiadomość z kodem weryfikującym, skopiuj go i podaj podczas logowania, a następnie dokonaj zmiany hasła."
+COM_USERS_RESET_REQUEST_LABEL="Wprowadź adres e-mail swojego konta. Kod weryfikacyjny zostanie wysłany do Ciebie. Po otrzymaniu kodu weryfikacyjnego będziesz mógł wybrać nowe hasło do swojego konta."
```

Rozważaliśmy to przed kilku laty i zdecydowaliśmy się na krótsze Wpisz, ale nie upieram się.
```suggestion
COM_USERS_RESET_REQUEST_LABEL="Wpisz adres e-mail swojego konta. Kod weryfikacyjny zostanie wysłany do Ciebie. Po otrzymaniu kodu weryfikacyjnego będziesz mógł wybrać nowe hasło do swojego konta."
```